README.md

```diff
@@ -53,6 +53,19 @@ using "Fix all" can be enabled by setting `unsafe-fixes = true` in your Ruff con
 See the [Ruff fix docs](https://docs.astral.sh/ruff/configuration/#fix-safety) for more details on how fix
 safety works.
 
+## Jupyter Notebook Support
+
+The extension has support for Jupyter Notebooks via the [Notebook Document Synchronization] capabilities of the Language
+Server Protocol which were added in 3.17. This has been implemented in `ruff-lsp` as of version `v0.0.43` which provides
+full support for all of the existing capabilities available to Python files in Jupyter Notebooks, including diagnostics,
+code actions, and formatting.
+
+This requires Ruff version `v0.1.3` or greater. Refer to the [Jupyter Notebook Discovery] section to learn how to configure
+Ruff to work with Jupyter Notebooks.
```

Comment:
Do they need to change this to get the VS Code extension to work? Doesn't it work without this, since we pass the file directly?

Reply:
Oh, right. I'll remove that part. Thanks for noticing!